Modules/Loadable/Markups/MRML/vtkMRMLMarkupsGenericStorageNode.h

```diff
@@ -0,0 +1,111 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Portions (c) Copyright Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  This file was originally developed by Johan Andruejol, Kitware Inc.
+  and was partially funded by Allen Institute.
+
+==============================================================================*/
+
+/// Markups Module MRML storage nodes
+///
+/// vtkMRMLMarkupsGenericStorageNode - MRML node for markups storage
+///
+/// vtkMRMLMarkupsGenericStorageNode nodes describe the markups storage
+/// node that allows to read/write fiducial point data from/to JSON.
+
+#ifndef __vtkMRMLMarkupsGenericStorageNode_h
+#define __vtkMRMLMarkupsGenericStorageNode_h
+
+// Markups includes
+#include "vtkSlicerMarkupsModuleMRMLExport.h"
+#include "vtkMRMLMarkupsStorageNode.h"
+
+class vtkMRMLMarkupsNode;
+
+/// \ingroup Slicer_QtModules_Markups
+class VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsGenericStorageNode
```

Comment:
In the class description, it may be sensible to describe how the key are organized in the translation map and how the key must be used. For example, by prepending `/` or surrounding with `/`